app/Http/Controllers/TalksController.php

```diff
@@ -25,19 +25,21 @@ class TalksController extends BaseController
         'slides.url' => 'Slides URL must contain a valid URL',
     ];
 
-    protected $sorting_talks = [
+    protected $sorting_style = [
         'date' => '',
         'alpha' => '',
     ];
 
     public function index()
     {
-        $talks = Auth::user()->talks()->active()->get();
-        list($talks, $sorting_talk) = $this->sortTalks($talks);
+        $talks = $this->sortTalks(
+            Auth::user()->talks()->active()->get()
+        );
+        $sorting_style = $this->sorting_style;
```

Comment:
You don't need a temp variable. Just pass `$this->sorting_style` directly to the `with` method.